File: Sierra-Code-Platoon/Week1/Day 1/algo-deaf-grandma.js
function hasLower(str){
    return str.toUpperCase() != str;
};


function deafGrandma() {

    var counter = 0;

    console.log("HEY KID!");

    const prompt = require('prompt-sync')();

    let input = prompt('');
    // console.log(`Hey there ${name}`)
    
    while(input == ""){
        input = prompt("WHAT!");

        if(hasLower(input)){
            console.log("SPEAK UP, KID!");
            input = "";
        }else if(hasLower(input) == false && input != ""){
            console.log("NO, NOT SINCE 1946!");
            input = "";
        }else if(input == "GOODBYE!" && counter == 2){
            console.log("LATER, SKATER!");
        }else if(input == "GOODBYE!"){
            counter++;
            console.log("LEAVING SO SOON?");
            input = "";
        }

    };
    
    
      

};

deafGrandma();

File: Sierra-Code-Platoon/Week1/Day 1/algo-deaf-grandma.py
def deaf_grandma():
    
    counter = 1
    ans = input("HEY KID!")
    
    while(counter != 3):
        ask = input("WHAT!")
        if(ask.isupper() and ask != "GOODBYE!"):
            print("NO, NOT SINCE 1946!")
        elif(ask.isupper() == False and len(ask)>0):
            print("SPEAK UP, KID!")
            if(ask == "GOODBYE!" and counter < 2):
                print("LEAVING SO SOON?")
                counter += 1
        elif(ask == "GOODBYE!" and counter < 2):
            print("LEAVING SO SOON?")
            counter += 1
        elif(ask == "GOODBYE!" and counter == 2):
            print("LATER, SKATER!")
            counter += 1
            

deaf_grandma()